components/addressbook/list.blade.php

```diff
@@ -10,6 +10,10 @@ class="list-group-item {{ ($customer->address_id == $address->address_id) ? 'lis
                         class="btn btn-outline-default"
                         href="{{ site_url('account/address', ['addressId' => $address->address_id]) }}"
                     >@lang('igniter.user::default.account.text_edit')</a>
+                    <a
+                        class="btn btn-outline-default"
+                        href="{{ site_url('account/address', ['defaultAddressId' => $address->address_id]) }}"
+                    >@lang('igniter.user::default.account.text_set_default')</a>
```

Comment:
should be igniter.user::default.text_set_default

Reply:
Also wrap the link in an @if ($customer->address_id != $address->address_id) so it only shows if its not the default.